flags/state.py

```diff
@@ -3,8 +3,16 @@
 
 def flag_state(flag_name, **kwargs):
     """ Return the value for the flag by passing kwargs to its conditions """
+
+    # If the request is given as a kwargs, and the FlagConditionsMiddleware is
+    # enabled, use the flag conditions attached to the request.
+    if hasattr(kwargs.get('request'), 'FLAG_CONDITIONS'):
```

Comment:
To avoid multiple places where `FLAG_CONDITIONS` is hardcoded, could this be stored as an attribute on the middleware class? See [this](https://github.com/cfpb/django-flags/compare/flags-middleware...chosak:flags-middleware-review#diff-6a2208f20b001e18840475536f7d4df2R11) for a suggested implementation.